testing/test_package_docs/ex/ex-library.html

```diff
@@ -92,7 +92,7 @@ <h1 class="title">
   <div class="col-xs-12 col-sm-9 col-md-8 main-content">
 
     <section class="desc markdown">
-      <p>a library. testing string escaping: <code>var s = 'a string'</code> &lt;cool&gt;</p>
+      <p>a library. testing string escaping: <code>var s = 'a string'</code> <cool></cool></p>
```

Comment:
The newer markdown library handles passing through html embedded in the markdown.